examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -39,6 +39,18 @@ apps: []
 #    protocol: "postgres"
 databases: []
 
+################################################################
+# Optional values if Database access is enabled.
+################################################################
+
+# Autodiscovery settings. Example:
+# aws_autodiscovery:
+# - types: ["rds"]
+#   regions: ["us-west-1"] # AWS regions to fetch databases from
+#   tags: # AWS database resource tags to match
+#     "*": "*"
+db_autodiscovery: []
```

Comment:
How about something like

```yaml
discovery:
  databases:
    aws:
    - types: ["rds"]
      regions: ["us-west-1"]
      tags:
        "*": "*"
```

This is a bit more nested structure but will allow us to have a place to extend this in the future to other things that can be auto-discovered like apps (e.g. on K8s), desktops or databases in other clouds.

WDYT @nicofff @webvictim?

Reply:
Yes, I agree this is a good way to go about it.